packages/dai-plugin-mcd/src/ManagedCdp.js

```diff
@@ -123,13 +107,11 @@ export default class ManagedCdp {
       this.getCollateralAmount(),
       this.getMinSafeCollateralAmount()
     ]);
-    return this.getCollateralAvailableSync();
+    return this.collateralAvailable;
   }
 
-  getCollateralAvailableSync() {
```

Comment:
I wonder if there's anyone relying on these `sync` functions? I feel like the right thing to do would be to deprecate them slowly (marking them now as deprecated and then removing them in a breaking release). What do you think?

Reply:
I don't think there's anyone relying on them. I added them pretty recently and we never documented them in any release notes, so I feel comfortable removing them directly and mentioning it in the next release.